internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4464,6 +4464,41 @@ func TestHandleGetInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetMempoolInfo(t *testing.T) {
+	t.Parallel()
```

Comment:
nit: I'd suggest adding a newline here for consistency with the rest of the tests.

Reply:
I'll wait for this to merge.